rpc/types.go

```diff
@@ -68,6 +68,26 @@ const (
 	EarliestBlockNumber = BlockNumber(0)
 )
 
+type RPCTransaction struct {
```

Comment:
This is exported because we have an external txpool client
I wonder if we should bring those network, txpool, admin, debug clients in
cc @mcortesi

Reply:
maybe... but not on this PR ;)